lib/jekyll-github-metadata/client.rb

```diff
@@ -59,9 +59,9 @@ def method_missing(method_name, *args, &block)
 
       def save_from_errors(default = false, &block)
         if block.arity == 1
-          block.call(@client)
+          yield(@client)
         else
-          block.call
+          yield
```

Comment:
Yeah, it's just confusing 😜 I *think* `yield` doesn't care about the arity of the block so maybe just simplify this to 

```ruby
def save_from_errors(default = false)
  yield @client
rescue # ...
# ...
end
```